### PR TITLE
Update pricing table for eks-for-enterprise

### PR DESCRIPTION
**Pull request by yorinasub17**
This implements the suggestion from https://gruntwork.atlassian.net/browse/WWW-119 to switch over the pricing model from EKS addon to being included in the Enterprise plan (as opposed to being a separate add-on to the subscription like CIS).

### CODE REVIEW

**Review by josh-padnick — APPROVED** [review]

**Comment by yorinasub17** [comment]
Thanks for review! Merging and releasing now.

File: pages/landing/enterprise-eks/pricing/index.html [hunk 1]
```diff
@@ -8,69 +8,143 @@
...
+      - name: Control plane
+        description: Support for configuring, deploying, and managing the EKS control plane.
+        plans: ["Standard", "Enterprise"]
+      - name: Worker nodes
+        description: "Support for deploying any of the three worker node types: self-managed ASG, EKS Managed Node Groups, and Fargate."
+        plans: ["Standard", "Enterprise"]
+      - name: Core services
+        description: Suite of core Kubernetes applications every EKS cluster should have, including AWS LoadBalancer controller, external-dns, cluster-autoscaler, and more.
+        plans: ["Standard", "Enterprise"]
+      - name: CI/CD
```
Comment (brikis98) [inline pages/landing/enterprise-eks/pricing/index.html:22]: Nit: perhaps "Basic CI/CD" to make it clearer there is an "advanced" version below, and to be consistent with the other "basic" and "advanced" options?

File: pages/landing/enterprise-eks/pricing/index.html [hunk 2]
```diff
@@ -8,69 +8,143 @@
...
+      - name: Basic monitoring
+        description: Support for metric and log aggregation to AWS CloudWatch.
+        plans: ["Standard", "Enterprise"]
+      - name: Basic deployment models
+        description: Support for rolling deployments where a configurable number of service Pods are replaced at a time.
+        plans: ["Standard", "Enterprise"]
+      - name: Basic networking
+        description: Support for AWS VPC based networking for Kubernetes Pods.
+        plans: ["Standard", "Enterprise"]
+      - name: Cluster auto scaling
```
Comment (brikis98) [inline pages/landing/enterprise-eks/pricing/index.html:34]: "Basic auto scaling"? The hover text provides the explanation of what that includes.

File: pages/landing/enterprise-eks/pricing/index.html [hunk 3]
```diff
@@ -8,69 +8,143 @@
 permalink: /eks-for-enterprise/pricing/
 
 categories:
-  - title: Feature
+  - title: EKS
```
Comment (brikis98) [inline pages/landing/enterprise-eks/pricing/index.html:11]: Note, the hero subtext says:

> Sign up now for early access to Amazon EKS for Enterprise! Early access subscribers will receive exclusive benefits and a significantly reduced subscription price.

But the page doesn't make it clear what the early access benefits are? The previous version of the page had "standard" vs "early access" but now it's "standard" vs "enterprise," so it's less clear.

File: pages/landing/enterprise-eks/pricing/index.html [hunk 4]
```diff
@@ -8,69 +8,143 @@
...
+      - name: Advanced monitoring
+        description: Support for metric and log aggregation through various 3rd party and in-cluster options, such as DataDog, Prometheus, Grafana, Elastic Stack, etc.
+        plans: ["Enterprise"]
+      - name: Advanced deployment models
+        description: Support for wide range of deployment models including canary, blue/green, automatic rollbacks, etc.
+        plans: ["Enterprise"]
+      - name: Advanced networking
+        description: Support for wide range of Pod networking options including Calico and Service Mesh.
+        plans: ["Enterprise"]
+      - name: Pod auto scaling
```
Comment (brikis98) [inline pages/landing/enterprise-eks/pricing/index.html:49]: Advanced auto scaling

**Review by brikis98 — COMMENTED** [review]
Thanks for putting this together Yori! I know it's merged already, but I left a few NITs that may make sense to do in a follow-up.

File: pages/landing/enterprise-eks/pricing/index.html (hunk 3 above)
Comment (yorinasub17) [reply pages/landing/enterprise-eks/pricing/index.html:11]: Switched to be the same as heroku landing page in #582.

File: pages/landing/enterprise-eks/pricing/index.html (hunk 4 above)
Comment (yorinasub17) [reply pages/landing/enterprise-eks/pricing/index.html:49]: I thought about the alternative way, but then realized that it's not as clean a split. E.g., the monitoring and networking options actually apply to both, as you get both cluster level logs + metrics and pod level. Similarly, service mesh arguably is service networking.

I think it's better to keep the table simple like it is now.

**Pull request merged**